app/rest/db-logging/src/main/java/io/syndesis/rest/dblogging/jaxrs/LogResource.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.rest.dblogging.jaxrs;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.syndesis.core.Json;
+import io.syndesis.core.KeyGenerator;
+import io.syndesis.jsondb.GetOptions;
+import io.syndesis.jsondb.JsonDB;
+import io.syndesis.rest.dblogging.jaxrs.model.Exchange;
+import io.syndesis.rest.dblogging.jaxrs.model.ExchangeStep;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+
+import static io.syndesis.rest.dblogging.jaxrs.JsonNodeSupport.fieldNames;
+import static io.syndesis.rest.dblogging.jaxrs.JsonNodeSupport.getLong;
+import static io.syndesis.rest.dblogging.jaxrs.JsonNodeSupport.getString;
+import static io.syndesis.rest.dblogging.jaxrs.JsonNodeSupport.removeBoolean;
+import static io.syndesis.rest.dblogging.jaxrs.JsonNodeSupport.removeLong;
+import static io.syndesis.rest.dblogging.jaxrs.JsonNodeSupport.removeString;
+
+/**
+ * Provides a JAXR interface to query stored integration logging from the DB.
+ */
+@Component
+public class LogResource {
+
+    private final JsonDB jsondb;
+
+    Set<String> EVENT_FIELDS_SKIP_LIST = new HashSet<>(Arrays.asList("id", "at", "logts"));
+
+    public LogResource(final JsonDB jsondb) {
+        this.jsondb = jsondb;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path(value = "/{integrationId}")
+    public List<Exchange> getLogs(
```

Comment:
Sorry, not sure whether I already asked: Why is this Jax RS endpoint when it is delegate to from the LogHandler in the rest api ?